tests/acceptance/blocking_adapter_test.py

```diff
@@ -476,13 +469,13 @@ def test(self):
         self.assertIsInstance(method, pika.spec.Basic.GetOk)
         self.assertEqual(method.delivery_tag, 1)
         self.assertFalse(method.redelivered)
-        self.assertEqual(method.exchange, _p3_as_bytes(''))
-        self.assertEqual(method.routing_key, q_name)
+        self.assertEqual(method.exchange, as_bytes(''))
+        self.assertEqual(method.routing_key, as_bytes(q_name))
```

Comment:
Yeah, forgot those callbacks receiving ``method``. That's a bit evil, I agree.

Strings are certainly not "UTF-8" in python3. They are unicode codepoints. The internal encoding varies on string to string basis and cannot be relied upon / used in any meaningful manner.

"Strings" in python2 are byte data with unknown encoding. Usually it's ASCII.